src/picongpu/include/fields/MaxwellSolver/None/NoSolver.hpp

```diff
@@ -34,26 +34,32 @@ namespace picongpu
     {
         using namespace PMacc;
 
+        /** Check Yee grid and time conditions
+         *
+         * This is a workaround that the condition check is only
+         * triggered if the current used solver is `NoSolver`
```

Comment:
note for later: even with staggered (but also possible with non-staggered e and b fields) the condition can be relaxed to: a particle with speed of light shall not be able to travel more than one cell (shortest edge).